### PR TITLE
Provide way to override homepage title and card description

### PR DESCRIPTION
**Pull request by maelle**
cf #957 

I'm unsure of where and how to best document this.

### CODE REVIEW

**Comment by maelle** [comment]
I added stuff related to "og:description" too. I took the freedom to edit pkgdown's own config file, but could remove that edit of course. :-)

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # pkgdown 1.3.0.9000 (development version)
 
+* One can override the title and description of the homepage via the `title` and `description` fields in the home section of config (#957, @maelle).
+
```
Comment (maelle) [inline NEWS.md]: btw this is the issue number not PR number.

**Comment by jayhesselberth** [comment]
@maelle have you completed work on this PR?

File: R/utils.r [hunk 2]
```diff
@@ -121,3 +121,7 @@ with_dir <- function(new, code) {
   on.exit(setwd(old))
   force(code)
 }
+
+uncranquote <- function(string) {
```
Comment (jayhesselberth) [inline R/utils.r]: `cran_unquote()` is probably better here.

Comment (maelle) [reply R/utils.r]: Done!

**Comment by maelle** [comment]
> @maelle have you completed work on this PR?

Yes I've stopped adding pieces and can now implement your feedback. :wink:

**Pull request merged**